base-template/contributors-guide.md

```diff
@@ -1,51 +1,250 @@
+1. Create a new branch for the work you want to do.
+   Make sure you give it an appropriate name, and include your username:
+
+   ```bash
+   > git checkout -b update-readme-username
+   ```
+
+Make your changes.
+When you have finished, save your changes, and create a pull request.
+
+Each branch and pull request must contain a single piece of work.
+For example, if you want to update a particular page and fix some typos across a lot of different pages, create a branch for the page update, make the changes, and do a pull request.
+Then create a new branch for the typos, and do a second pull request.
+This makes it easier for you to track your own work, and also for reviewers looking at your pull requests.
+
+#### Creating a pull request (PR)
+
+1. Save your changes in your text editor.
+1. In your terminal, add the changes to your commit:
+
+   ```bash
+   > git add .
+   ```
+
+   Don't forget the period at the end of this command!
+1. Commit the changes, and provide a short description of what's in the commit:
+
+   ```bash
+   > git commit -m "Updated README file"
+   ```
+
+1. Push the changes to your fork:
+
+   ```bash
+   git push
+   ```
+
+1. In your browser, go to your fork on github.
+   The URL is in the form `github.com/<your_username>/incubator`, or you can get to it by clicking on your user image from the main github page, and selecting `Your repositories`.
+   In the banner at the top of the page, click `Compare and Create Pull Request`.
+1. Give your PR a title, and write a description of the changes.
+   Link any relevant issues, and tag people you want to review your changes.
+   If you intend to add more commits to your PR, mark your PR as a draft so that reviewers know you have not finished.
 
 ## Choosing something to work on
 
-Compellingly iterate premium testing procedures through effective web services. Collaboratively recaptiualize client-based leadership skills rather than adaptive initiatives. Distinctively synthesize customer directed markets after worldwide portals. Appropriately exploit revolutionary e-services without excellent content. Assertively unleash clicks-and-mortar leadership skills with 2.0 action items.
+This section discusses how to find something to work on within the Good Docs Project.
 
 ### Issues
 
-Assertively enhance functional solutions for synergistic opportunities. Proactively expedite cross-media synergy whereas cutting-edge services. Appropriately embrace principle-centered leadership via leveraged infomediaries. Competently negotiate top-line products through 24/365 experiences. Professionally matrix goal-oriented processes and bleeding-edge experiences.
+Like most open source projects, there are always tasks to do.
+For the list of issues, see [https://github.com/thegooddocsproject/templates/issues](The Good Docs Project template issues).
+Look for issues with a `good first issue` tag if you want something small to start on.
```

Comment:
💯 like this idea, just not sure we are tagging things appropriately and we should link to the tags.

Reply:
I checked, there are a few "good first issue" tags in the repo, but I agree that we should probably review our practices around this.